modules/andes-core/broker/src/main/java/org/dna/mqtt/moquette/messaging/spi/impl/ProtocolProcessor.java

```diff
@@ -205,12 +205,13 @@ void processConnect(ServerChannel session, ConnectMessage msg) {
 
         //Handle will flag
         if (msg.isWillFlag()) {
-            AbstractMessage.QOSType willQos = AbstractMessage.QOSType.values()[msg.getWillQos()];
-            byte[] willPayload = msg.getWillMessage().getBytes();
-            ByteBuffer bb = (ByteBuffer) ByteBuffer.allocate(willPayload.length).put(willPayload).flip();
-            PublishEvent pubEvt = new PublishEvent(msg.getWillTopic(), willQos,
-                    bb, msg.isWillRetain(), msg.getClientID(), session);
-            processPublish(pubEvt);
+            log.warn("Andes does not support last will operation");
+//            AbstractMessage.QOSType willQos = AbstractMessage.QOSType.values()[msg.getWillQos()];
+//            byte[] willPayload = msg.getWillMessage().getBytes();
+//            ByteBuffer bb = (ByteBuffer) ByteBuffer.allocate(willPayload.length).put(willPayload).flip();
+//            PublishEvent pubEvt = new PublishEvent(msg.getWillTopic(), willQos,
+//                    bb, msg.isWillRetain(), msg.getClientID(), session);
+//            processPublish(pubEvt);
```

Comment:
Do we need to keep commented out code? We can get them back from GIT later if we want to.